engine/src/main/resources/templates/Inventory/validationErrorEmailBody.html

```diff
@@ -0,0 +1,40 @@
+{#include Inventory/insightsEmailBody}
+{#content-title}Validation error Instant Notification - {action.timestamp.toStringFormat()}{/content-title}
+{#content-body}
+<tr>
+    <td class="rh-content__block">
+        <table class="rh-data-table rh-m-bordered">
+            <thead>
+            <tr>
+                <th>System</th>
+                <th>Error</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td>
+                    {#if action.events[0].payload.host_id is not null}<a href="{environment.url}/insights/inventory/{action.events[0].payload.host_id}" target="_blank">{action.events[0].payload.display_name}</a>{#else}{action.events[0].payload.display_name}{/if}
```

Comment:
This should actually be `!=` instead of `is not`
```suggestion
                    {#if action.events[0].payload.host_id != null}<a href="{environment.url}/insights/inventory/{action.events[0].payload.host_id}" target="_blank">{action.events[0].payload.display_name}</a>{#else}{action.events[0].payload.display_name}{/if}
```